File: src/Book.java
import java.util.Random;

public class Book {
    private int id;
    private String title;
    private String author;
    private boolean rented;

    Random ran = new Random();

    public Book(String title, String author) {
        this.id=ran.nextInt(1,1000);
        this.title=title;
        this.author=author;
        this.rented=false;
    }
    public Book(int id, String title, String author) {
        this.id=id;
        this.title=title;
        this.author=author;
        this.rented=true;
    }

    public Book(String title, String author, boolean rented) {
        this.id=ran.nextInt(1,1000);
        this.title=title;
        this.author=author;
        this.rented=rented;
    }

    public int getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public String getAuthor() {
        return author;
    }

    public boolean isRented() {
        return rented;
    }

    public void setRented(boolean rented) {
        this.rented = rented;
    }
}

File: book.py
import random

class Book:
    def __init__(self, title, author, rented=False):
        self.id = random.randint(1, 1000)
        self.title = title
        self.author = author
        self.rented = rented

    def getId(self):
        return self.id

    def getTitle(self):
        return self.title

    def getAuthor(self):
        return self.author

    def isRented(self):
        return self.rented

    def setRented(self, rented):
        self.rented = rented
